deprecate chat order field (#687)
livechat/livechat-public-docs: content/messaging/agent-chat-api/changelog/index.mdx

- There're new fields `previous_thread_id` and `next_thread_id` in the [**Thread**](/messaging/agent-chat-api/v3.2/#threads) data structure.
- Fields `order` and `timestamp` in the [**Thread**](/messaging/agent-chat-api/v3.2/#threads) data structure were replaced with the new field, `created_at` (date & time in microseconds in UTC).
- The **Get Chat Threads** method evolved into **Get Chat** ([Web](/messaging/agent-chat-api/v3.2/#get-chat) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-chat)). It now allows to retrieve a chat containing a particular thread. It also changed its shape; it no longer contains the `order` field and the `chat` key.

> - `order` in `chat` object has been removed from the responses of **Login** and **List Chats** methods

Reviewer: ```suggestion
- The `order` field in the `chat` object was removed from the responses of the **Login** and **List Chats** methods.
```
